src/views/CreateEventView.vue

```diff
@@ -88,17 +106,43 @@ export default {
         description: String,
         whatToBring: String,
       */
+      imageUploaded: 0,
       event: {},
       error: '',
     };
   },
   methods: {
+    convertImage(file) {
+      return new Promise((resolve, reject) => {
+        const reader = new FileReader();
+        reader.readAsDataURL(file);
+        reader.onload = () => resolve(reader.result);
+        reader.onerror = error => reject(error);
+      });
+    },
+    updateEventImage(event) {
```

Comment:
Is it possible/worth it to unit test either of these functions?

Reply:
That is a great question